### PR TITLE
#30 DatasetEditManager unit test

### PR DESCRIPTION
**Pull request by sergiuilie**

### CODE REVIEW

**Comment by sergiuilie** [comment]
@shciubrcom if you squash, do not forget to put the issue ID

File: src/__mocks__/vscode.ts [hunk 1]
```diff
@@ -29,6 +27,7 @@ export enum TreeItemCollapsibleState {
     Expanded = 2,
 }
 
+// tslint:disable-next-line: no-namespace
```
Comment (shciubrcom) [inline src/__mocks__/vscode.ts:30]: `// tslint:disable: max-classes-per-file no-namespace` should be reeped and `// tslint:disable-next-line: no-namespace` can be removed

File: src/__mocks__/vscode.ts [hunk 2]
```diff
@@ -149,8 +217,32 @@ export class TreeItem {
     }
 }
 
+// tslint:disable-next-line: max-classes-per-file
+// export class EventEmitter {
```
Comment (shciubrcom) [inline src/__mocks__/vscode.ts:221]: Commented code should be removed

**Review by shciubrcom — CHANGES_REQUESTED** [review]
Seems that some tests miss asserts

**Pull request merged**